docs/man_pages/project/testing/publish.md

```diff
@@ -0,0 +1,43 @@
+publish
+==========
+
+<% if((isConsole && isMacOS) || isHtml) { %>
+Usage | Synopsis
+---|---
+General | `$ tns publish ios`
+
+Builds the project and publishes it to an Application Store.
+
+<% if(isHtml) { %>
+### Command Limitations
+
+* You can currently run `$ tns publish` only on OS X systems.
```

Comment:
The idea behind this is, that in the very near future the publish command will become applicable to android also, and thus, be applicable to Windows and Linux OS.
For the time being I'll take your comment into account though - when the time comes and the `$ tns publish android` command is implemented, the help for this command will change either way